app/src/main/AndroidManifest.xml

```diff
@@ -331,11 +331,6 @@
                     android:scheme="package"/>
             </intent-filter>
         </receiver>
-        <receiver android:name="de.danoeh.antennapod.core.service.playback.MediaButtonIntentReceiver">
-            <intent-filter>
-                <action android:name="android.intent.action.MEDIA_BUTTON" />
-            </intent-filter>
-        </receiver>
```

Comment:
Haven't run your code yet, but how are Bluetooth devices and maybe Android Wear able to control the playback without this?
-> http://developer.android.com/training/managing-audio/volume-playback.html#PlaybackControls